client/src/app/pages/applications/analysis-wizard/targets.ts

```diff
@@ -193,9 +194,10 @@ export const transformationTargets: ITransformationTargets[] = [
     label: "Azure",
     description:
       "Upgrade your Java application so it can be deployed in different flavors of Azure.",
-    options: isPropietaryAllowed
-      ? ["azure-appservice", "azure-aks"]
-      : ["azure-appservice"],
+    options:
+      APP_BRAND === BrandType.Konveyor
+        ? ["azure-appservice", "azure-aks"]
```

Comment:
This isn't really a comment about this change, but what's the deal here? users aren't allowed to analyze AKS apps if they're using the Red Hat branded version of MTA?

Reply:
Yeah @gildub Can probably speak to this more but this is a recent change that adds "proprietary" targets for upstream images only.

Reply:
@mturley, let's chat about that so I can explain what's happening.

Reply:
Sounds good, I'll resolve this thread and we can discuss on slack